.github/workflows/e2e-olm/e2e-tests-olm.yml

```diff
@@ -0,0 +1,79 @@
+name: e2e-tests-olm
+description: Runs E2E tests against the OLM bundle
+on:
+  pull_request:
+
+    branches:
+    - v1alpha1
+
+env:
+  KIND_VERSION: "0.15.0"
+  GO111MODULE: "on"
+  OPERATOR_IMG: "quay.io/sustainable_computing_io/kepler-operator"
+  KUBECONFIG: /tmp/.kube/config
```

Comment:
shouldn't this change depending on the `PROVIDER`? 
FYI local-dev-cluster scripts now copy <provider>/kubconfig to ~/.kube/config so this may not be even required.